src/Umbraco.Abstractions/UmbracoContextReference.cs

```diff
@@ -20,9 +20,8 @@ public class UmbracoContextReference : IDisposable //fixme - should we inherit f
         /// <summary>
         /// Initializes a new instance of the <see cref="UmbracoContextReference"/> class.
         /// </summary>
-        internal UmbracoContextReference(IUmbracoContext umbracoContext, bool isRoot, IUmbracoContextAccessor umbracoContextAccessor)
+        internal UmbracoContextReference(bool isRoot, IUmbracoContextAccessor umbracoContextAccessor)
```

Comment:
This change i'm just unsure of due to how UmbracoContextReference and IUmbracoContextFactory works. This reference should return the literal reference of IUmbracoContext not the result of the accessor since there's no assurance that the accessor is returning the correct reference. Thoughts?

Reply:
I can change it back to inject the `IUmbracoContext`..